chat-widget/src/components/webchatcontainerstateful/common/mockchatsdk.ts

```diff
@@ -71,7 +71,7 @@ export class MockChatSDK {
     public getLiveChatConfig() {      
         return {
             LiveWSAndLiveChatEngJoin: {
-                msdyn_postconversationsurveyenable: "true",
+                msdyn_postconversationsurveyenable: "false",
```

Comment:
`DesignerChatSDK` is inheritng from `MockChatSDK`. I'd override `getLiveChatConfig()` from `DesignerChatSDK` in case we'll be breaking some changes with other components using `MockChatSDK`.

Reply:
Done. Thanks!